ouroboros-network/src/Ouroboros/Network/PeerSelection/State/KnownPeers.hs

```diff
@@ -338,6 +359,12 @@ setTepidFlag :: Ord peeraddr
              -> KnownPeers peeraddr
 setTepidFlag = setTepidFlag' True
 
+setSuccessfulConnectionFlag :: Ord peeraddr
+                            => peeraddr
+                            -> KnownPeers peeraddr
+                            -> KnownPeers peeraddr
+setSuccessfulConnectionFlag = setSuccessfulConnectionFlag' True
```

Comment:
Why not inline `setSuccessfulConnectionFlag'` here? It's not used anywhere else.

Reply:
I thought I was following the style of the other functions, but realise now its not it